Kickstarter-iOS/ViewModels/AppDelegateViewModelTests.swift

```diff
@@ -2348,6 +2348,31 @@ final class AppDelegateViewModelTests: TestCase {
       self.goToLandingPage.assertDidNotEmitValue()
     }
   }
+
+  func testDeeplink_DoesNotActivateIf_GoToLandingPageEmits() {
+    let optimizelyClient = MockOptimizelyClient()
+      |> \.experiments .~
+      [OptimizelyExperiment.Key.nativeOnboarding.rawValue: OptimizelyExperiment.Variant.variant1.rawValue]
+
+    let userDefaults = MockKeyValueStore()
+      |> \.hasSeenLandingPage .~ false
+
+    withEnvironment(currentUser: nil, optimizelyClient: optimizelyClient, userDefaults: userDefaults) {
+      self.goToLandingPage.assertDidNotEmitValue()
+
+      self.vm.inputs.applicationDidFinishLaunching(application: UIApplication.shared, launchOptions: nil)
+      self.vm.inputs.didUpdateOptimizelyClient(MockOptimizelyClient())
+
+      _ = self.vm.inputs.applicationOpenUrl(
```

Comment:
Can we actually run this test with a project deeplink? The output that was being triggered during the bug was `presentViewController`, which first dismisses any previously presented VC.